lib/helpers/backend/googleBooksSearch.ts

```diff
@@ -0,0 +1,22 @@
+import { GoogleBook } from "../../../common/types";
+import { google, books_v1 } from "googleapis";
+import { GaxiosResponse } from "gaxios";
+
+const book = google.books({
+  version: "v1",
+});
+
+export const getGoogleBooksData = async (
```

Comment:
Thoughts on putting this module in the services folder? Feels like more of a service to me than a collection of helper functions.

Reply:
Also, this module needs to be documented in the spec

Reply:
Done in latest commit.